### ISSUE TITLE
Va-Slider does not work as expected with screen readers (NVDA and Windows Narrator)

### ISSUE THREAD

**Issue by iangoodall** [epicmaxco/vuestic-ui#4392]
**Vuestic-ui version:** 1.10.2

Hi Vuestic team! During some accessibility testing I came across the following issue that I'd like to highlight.

### Description

  - No announcement of details of the slider in screen reader (Windows Narrator or NVDA)
  - [Like the W3 example](https://www.w3.org/WAI/ARIA/apg/patterns/slider-multithumb/examples/slider-multithumb/), details of the current value and change in values should be read by the screen reader.

### Reproduction

See [this playground](https://ui.vuestic.dev/play#eNp9UstOwzAQ/BXLJ5DaRhWPQylIgHqAA6AWccEcrGTbuDi2Za9Dpar/zjppQkGop+zO7HpnRtnyW+dGdQQ+4dOQe+XwRhjYOOuRFbCUUSPbCsNYIVGenLY1Yx4wetN1jNVSR5iw9/OLAbu8+Bi0+O4qfXfUpWqa9QeoQaiclgjUMTYtxyzXMoRrwWs5LMeCNzhjc2lWwIJWBfhmMivH7cqbXPQoCRhWtgDd7JMUwfc603oqM1qaZgdHheEDrqpkdFhJN1oHayiExpHYE0HwSeeRHo4QUOXDqBIseInowiTLokoBJmpUQJ11YzZiVkll6OVWzY5ioJsYcmuWavXnYm4rpzT4Z4fKmt+Xpdb267HB0EfYp0s7JeSf/+DrsGklvngI4GuKo+dQ+hVgS88WT7Chuicpw6hp+gg5h2B1TBrbsbtoCpJ9MNeofWgCVGb1GmYbBBM6U0lo91fsU70/Yv1H7tnorE9x9w0As+H/) with the basic set up of a range slider.

1. Open up Windows Narrator of the [latest version of NVDA](https://www.nvaccess.org/download/)
2. Tab into the Slider
3. Note that in both screen readers, they are silent and provide no information to the user.

I hope this is enough information for you to at least reproduce the issue. Happy for you to reply if you need anything else. Thank you.
